commands/blocked.js

```diff
@@ -0,0 +1,33 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { MessageActionRow, MessageButton } = require('discord.js');
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('blocked')
+        .setDescription('Gives information on what to do when you are blocked')
```

Comment:
Missing a comma at the end of this, Maybe also change the text to be "when you (or another user) is blocked"?

Reply:
Done.